flight/Modules/Actuator/actuator.c

```diff
@@ -691,7 +589,7 @@ static bool set_channel(uint8_t mixer_channel, float value, const ActuatorSettin
                             }
 			}
 			PIOS_Servo_Set(mixer_channel,
-							buzzOn?actuatorSettings->ChannelMax[mixer_channel]:actuatorSettings->ChannelMin[mixer_channel]);
+							buzzOn?actuatorSettings.ChannelMax[mixer_channel]:actuatorSettings.ChannelMin[mixer_channel]);
```

Comment:
I've pushed another change with uncrustify applied to the file with proposed settings that match most of the flight mode style.  Perhaps we consider that?  If we don't want to reindent that I can pull that and just reindent those two lines.